src/me/me.v6.service.js

```diff
@@ -23,7 +23,12 @@ angular.module("ovh-api-services").service("OvhApiMeV6", function ($resource, $c
             method: "GET",
             url: "/me/consumption/usage/history",
             isArray: true
-        }
+        },
+        supportLevel: {
+          method: "GET",
+          url: "/me/supportLevel",
+          isArray: false
```

Comment:
No need for this, `isArray` is `false` by default

Reply:
Explicit version has been preferred in that case :)